core/shared/src/main/scala/zio/metrics/Metric.scala

```diff
@@ -353,19 +401,23 @@ object Metric {
 
         override private[zio] val unsafe: UnsafeAPI =
           new UnsafeAPI {
-            def update(in: uz.In, extraTags: Set[MetricLabel] = Set.empty)(implicit unsafe: Unsafe): Unit = {
+            def update(in: uz.In, extraTags: Set[MetricLabel], description: Option[String])(implicit
```

Comment:
We should keep the default parameter for `extraTags` here and add one for `description`.